spec/unit/typesafe/config/path_spec.rb

```diff
@@ -0,0 +1,259 @@
+require 'spec_helper'
+require 'hocon'
+require 'test_utils'
+
+
+describe Hocon::Impl::Path do
+  Path = Hocon::Impl::Path
```

Comment:
This looks a little weird to me ... doesn't seem problematic, but ... is this a ruby convention?